Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/PressableButtonInspector.cs

```diff
@@ -231,7 +229,7 @@ private float DrawPlaneAndHandle(Vector3[] vertices, Vector3 halfExtents, float
             {
                 float handleSize = HandleUtility.GetHandleSize(vertices[1]) * 0.15f;
 
-                Vector3 dir = (touchable != null) ? -1.0f * touchable.LocalForward : Vector3.forward;
+                Vector3 dir = (touchable != null) ? touchable.LocalPressDirection : Vector3.forward;
```

Comment:
Since  we are  now negating LocalForward to be LocalPressDirection, will this mess up people who have created custom pressable buttons, or other custom elements that have near interaction touchables? Say in our next release, if those assets still have LocalForward to be pointing at (0, 0, 1), will they no longer be able to press buttons until they re-configure the buttons? If yes, let's try to figure out a plan for migration of our customers. A surprising number of customers create copies of our prefabs and then edit / customize them, and the breaks on upgrade might make them reluctant to upgrade and get all the goodness we are adding

Reply:
For existing NearInteractionTouchables, this should still work.  No migration plan should be required.

Whatever they have set as their local forward direction on their own prefabs (or prefab variants of MRTK buttons) will be negated into LocalPressDirection.  So the value produced by this line will be identical now as it was prior to my change.  The existing button prefabs in MRTK already have non-default localForward, and they function correctly.  Therefore the MRTK buttons are equivalent to the upgrade scenario, so I think this should satisfy testing.